### PR TITLE
README file creation

### PR DESCRIPTION
**Pull request by RuslanPleskun**

### CODE REVIEW

**Review by andrii-s-shpitser-lpnu-ua — COMMENTED** [review]
develop branch required instead of master

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# __ChromeDragonGame__
+This repo has been created to implement one of the most famous game _ChromeDragon_ and after a while you will be able to try it out here.
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md]: add details about yourself and full task details

**Comment by RuslanPleskun** [comment]
> develop branch required instead of master

My bad. Just updated with correct branch. Could you please check it again?

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
> > develop branch required instead of master
> 
> My bad. Just updated with correct branch. Could you please check it again?

master can be removed. And task details still not added to readme.

**Review by andrii-s-shpitser-lpnu-ua — COMMENTED** [review]
Please remove master branch and make develop as default.

**Comment by RuslanPleskun** [comment]
> Please remove master branch and make develop as default.

Main branch has been removed, develop branch was set as default and README file has been updated with details about myself and full task details. Please check one more time if now everything is good.

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**